Modules/_queuemodule.c

```diff
@@ -0,0 +1,422 @@
+{
+    PyObject *r;
+    if (self->lst_pos == PyList_GET_SIZE(self->lst))
+        r = Py_True;
+    else
+        r = Py_False;
+    Py_INCREF(r);
+    return r;
+}
+
+/*[clinic input]
+_queue.SimpleQueue.qsize
+
+Return the approximate size of the queue (not reliable!).
+[clinic start generated code]*/
+
+static PyObject *
+_queue_SimpleQueue_qsize_impl(simplequeueobject *self)
+/*[clinic end generated code: output=4c6f52e316a53f60 input=76444ac3249e90c5]*/
+{
+    return PyLong_FromSsize_t(PyList_GET_SIZE(self->lst) - self->lst_pos);
+}
+
+static PyObject *
+simplequeue_sizeof(simplequeueobject *self, void *unused)
+{
+    Py_ssize_t res;
+
+    res = _PyObject_SIZE(Py_TYPE(self));
+    res += _PySys_GetSizeOf(self->lst);
+    return PyLong_FromSsize_t(res);
+}
+
+
+#include "clinic/_queuemodule.c.h"
+
+
+static PyMethodDef simplequeue_methods[] = {
+    _QUEUE_SIMPLEQUEUE_EMPTY_METHODDEF
+    _QUEUE_SIMPLEQUEUE_GET_METHODDEF
+    _QUEUE_SIMPLEQUEUE_GET_NOWAIT_METHODDEF
+    _QUEUE_SIMPLEQUEUE_PUT_METHODDEF
+    _QUEUE_SIMPLEQUEUE_PUT_NOWAIT_METHODDEF
+    _QUEUE_SIMPLEQUEUE_QSIZE_METHODDEF
+    {"__sizeof__", (PyCFunction) simplequeue_sizeof, METH_NOARGS, NULL},
+    {NULL,           NULL}              /* sentinel */
+};
+
+
+PyTypeObject PySimpleQueueType = {
+    PyVarObject_HEAD_INIT(NULL, 0)
+    "_queue.SimpleQueue",               /*tp_name*/
+    sizeof(simplequeueobject),          /*tp_size*/
+    0,                                  /*tp_itemsize*/
+    /* methods */
+    (destructor)simplequeue_dealloc,          /*tp_dealloc*/
+    0,                                  /*tp_print*/
+    0,                                  /*tp_getattr*/
+    0,                                  /*tp_setattr*/
+    0,                                  /*tp_reserved*/
+    0,                                  /*tp_repr*/
+    0,                                  /*tp_as_number*/
+    0,                                  /*tp_as_sequence*/
+    0,                                  /*tp_as_mapping*/
+    0,                                  /*tp_hash*/
+    0,                                  /*tp_call*/
+    0,                                  /*tp_str*/
+    0,                                  /*tp_getattro*/
+    0,                                  /*tp_setattro*/
+    0,                                  /*tp_as_buffer*/
+    Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE
+        | Py_TPFLAGS_HAVE_GC,           /* tp_flags */
+    _queue_SimpleQueue___init____doc__, /*tp_doc*/
+    (traverseproc)simplequeue_traverse, /*tp_traverse*/
+    0,                                  /*tp_clear*/
+    0,                                  /*tp_richcompare*/
+    offsetof(simplequeueobject, weakreflist), /*tp_weaklistoffset*/
+    0,                                  /*tp_iter*/
+    0,                                  /*tp_iternext*/
+    simplequeue_methods,                /*tp_methods*/
+    0,                                  /* tp_members */
```

Comment:
Inconsistent use of spaces in comments.

Reply:
I will happily let others reformat if they're irritated by inconsistent spaces in comments.